CHANGELOG.md

```diff
@@ -2666,6 +2666,7 @@
 - Fixing empty null lists throwing explode_outer for a loop. ([#7649](https://github.com/rapidsai/cudf/pull/7649)) [@hyperbolic2346](https://github.com/hyperbolic2346)
 - Fix internal compiler error during JNI Docker build ([#7645](https://github.com/rapidsai/cudf/pull/7645)) [@jlowe](https://github.com/jlowe)
 - Fix Debug build break with device_uvectors in grouped_rolling.cu ([#7633](https://github.com/rapidsai/cudf/pull/7633)) [@mythrocks](https://github.com/mythrocks)
+- Parquet reader: Fix issue when using skip_rows on non-nested columns containing nulls ([#7627](https://github.com/rapidsai/cudf/pull/7627)) [@nvdbaranec](https://github.com/nvdbaranec)
```

Comment:
Can we avoid making unrelated CHANGELOG entries? I assume this might have been due to a conflict in forward-merging, or something like that?

edit: this PR definitely needs a rebase on 23.06. The commit history is way off.

Reply:
Sync'd offline and we decided this is fixed via squash+merge.